### ISSUE TITLE
no-empty-rulesets does not catch rulesets containing only comments

### ISSUE THREAD

**Issue by royaldark** [sasstools/sass-lint#968]
What version of Sass Lint are you using?
1.10.2

With `no-empty-rulesets` enabled, rulesets containing only comments are not considered empty, and therefore do not trigger the rule. I would argue rulesets like the following are definitely empty:

```scss
.baz {
  // Here is a comment
  // These comments often contain rules that were commented out
  // width: 4px;
}
```

Does this seem like a reasonable assumption?

I worked up a commit that changes the behavior to fix this if there's any interest.

**Comment by DanPurdy** [comment]
RIghtly or wrongly I chose to ignore this while we didn't have the ability to disable rules via comments but now that we do I think it's perfectly reasonable to believe that this should still fail as the comments aren't being pumped out into CSS. 

There's an argument for `/* */` style comments not throwing here as technically they do get added to the resulting stylesheet but arguably the rule is still empty. What do you think?

Also as with all our bug issues PR's are welcome!

**Comment by royaldark** [comment]
I guess I'm not sure why you'd ever want to preserve an empty block, even with comments. Similar tools like CSSLint seem to agree: https://github.com/CSSLint/csslint/wiki/Disallow-empty-rules

I'll submit a PR in the next few days.
